bower.json

```diff
@@ -1,15 +1,14 @@
 {
   "name": "cfgov-sheer-templates",
   "description": "Sheer templates for consumerfinance.gov",
-  "version": "0.3.1",
+  "version": "1.0.0",
```

Comment:
I'm guessing it's because bower uses GH as a data store. If you listed `1.2.0` as the latest version in bower.json but there was no `1.2.0` tag at GH, `bower install cfgov-sheer-templates` would throw an error. Given how common it is to forget to push tags to GH, the error would surface constantly and break lots of people's builds.

Instead, bower gets a list of a component's available versions by connecting directly to GH and checking the tags. Because it's talking to GH and ignoring the version number in bower.json, they officially told people to stop using the version property to avoid confusion.

npm, on the other hand, maintains a massive database that stores an archive of all published package versions and doesn't talk to GH.